File: Day  Two/Python/student_grade.py

score_one = 30
score_two = 55
score_three =40




def calculating_the_average_score_grade (score_one, score_two, score_three ):
    averageResult =  score_one + score_two + score_three / 3 

    if (90 <= averageResult <=100):
        print("Letter Grade A")

    elif (80<= averageResult <90):
        print ("Letter Grade B")

    elif (70 <= averageResult < 80):
        print("Letter Grade C")

    elif (60 <= averageResult <= 70):
        print ("Letter Grade D")
    

    elif (0 <= averageResult <= 60):
        print ("Letter Grade F")

    return averageResult 

    

print (calculating_the_average_score_grade (score_one, score_two, score_three))
 

File: Day  Two/Java Script/After Break/StudentGrade.js

let scoreOne = 30;
let scoreTwo = 40;
let scoreThree = 50;



function calculatingTheAverageScoreGrade ( scoreOne, scoreTwo, scoreThree)
    const averageResult =  scoreOne + scoreTwo + scoreThree / 3;
    return averageResult;
    
    }

        if (averageResult <=90 && averageResult <=100) {
            console.log("Letter Grade A");
        }

        else if (averageResult <=80 && averageResult < 90) {
            console.log("Letter Grade B");
        }
        else if (averageResult <=70 && averageResult < 80) {
            console.log("Letter Grade C"); 
        }
        else if (averageResult <= 60 && averageResult < 70) {
            console.log("Letter Grade D");
        }
        
        else if (averageResult <= 90 && averageResult <60) {
            console.log("Letter Grade F");
        
       }
    }
}





